content/docs/latest/spiffe-about/overview.md

```diff
@@ -96,19 +96,20 @@ SPIFFE enjoys a broad ecosystem, with many software projects, products, and plat
 
 ### Commercial Software that Works with SPIFFE
 
-| Software               | X.509 SVID | JWT SVID | SPIFFE Authentication | Workload API |
```

Comment:
While we change the table, should we add links to the corresponding documentation?

- GCP Workload Identity Federation: https://cloud.google.com/iam/docs/workload-identity-federation
- AWS IAM Roles Anywhere: https://docs.aws.amazon.com/rolesanywhere/latest/userguide/introduction.html
- Graymatter.io: Only found this: https://greymatter.gitbook.io/grey-matter-documentation/usage/fabric/security/spire
- VMware Secrets Manager: Maybe https://vsecm.com/documentation/architecture/overview/ ?

Reply:
Yes - I think this is a great idea! The more pointers we can give people the better.

Reply:
Do you have a preference re: linking from the name of the software versus presenting the bare link in a new column?

Reply:
No preference from my side. Eventually comes down how it's rendered and if bare links mess up the table ..